.devcontainer/Dockerfile

```diff
@@ -30,6 +30,12 @@ WORKDIR /polkadot-binary
 RUN wget "https://github.com/paritytech/polkadot/releases/download/v0.9.22/polkadot" && \
     chmod +x polkadot
 
+
+# # Add Basilisk node
+WORKDIR /Basilisk-node
+RUN wget "https://github.com/galacticcouncil/Basilisk-node/releases/download/v8.0.0/basilisk" && \
+    chmod +x basilisk
```

Comment:
@cor Would love to!
We can do this here. But not for `/workspaces/Basilisk-node/target/release`, because the current polkadot launcher configuration expects it with an upper case.

Or we change it here as well, then we're free to go:
https://github.com/ComposableFi/composable/blob/main/scripts/polkadot-launch/composable_and_basilisk.json#L67

Reply:
Capitalizing binaries is very Windows... I know my preference.

Reply:
@Dom-Roth please change it there aswell then :-)